frontend/src/metabase/dashboard/components/PublicLinkPopover/PublicLinkCopyPanel.styled.tsx

```diff
@@ -15,7 +14,7 @@ export const PublicLinkCopyButton = styled(CopyButton)`
 `;
 
 export const RemoveLinkAnchor = styled(Anchor)<
-  AnchorProps & HTMLAttributes<HTMLAnchorElement>
+  ComponentProps<typeof Anchor<ElementType>> & HTMLAttributes<HTMLAnchorElement>
```

Comment:
This is the reason `@typescript-eslint/eslint-plugin` needs to be upgraded, because it was always complaining about this syntax.

Reply:
Nice! this is a much better way to get `component` other than just specifying it in the styled props.